app/questionnaire/rules.py

```diff
@@ -269,12 +320,22 @@ def evaluate_when_rules(
     return True
 
 
-def get_answer_store_value(answer_id, answer_store, schema, routing_path=None):
+def get_answer_store_value(
```

Comment:
This could do with being better named, due to a lot of other similarly named methods. Maybe get_answer_on_routing_path?

Reply:
I've gone with `get_answer_value`.